scripts/linux/buildManifest.sh

```diff
@@ -159,11 +157,11 @@ cat $manifest
 echo "Done Building And Pushing Docker Images"
```

Comment:
Can we move this log down below the push? I was the one that caused this problem, and I think this log contributed to my mistake.